src/Common/Event_Automator/Admin/Tabs/Integrations.php

```diff
@@ -0,0 +1,106 @@
+<?php
+
+namespace TEC\Common\Event_Automator\Admin\Tabs;
```

Comment:
```suggestion
namespace TEC\Event_Automator\Admin\Tabs;
```
Keep namespacing the same.